frontend/src/metabase/query_builder/components/notebook/steps/ClauseStep/ClauseStep.tsx

```diff
@@ -88,3 +128,38 @@ export const ClauseStep = <T,>({
     </NotebookCell>
   );
 };
+
+interface ClauseStepItemProps {
+  id: string;
+  children: ReactNode;
+}
+
+function ClauseStepItem({ id, children }: ClauseStepItemProps) {
+  const { attributes, listeners, setNodeRef, transform, transition } =
+    useSortable({
+      id,
+      animateLayoutChanges: () => false,
```

Comment:
Disable animation after drag-n-drop because we don't have a stable ID

Reply:
This comment deserves to live in code